### PR TITLE
Fixed pose skipping if no neck

### LINKED ISSUE #67: Poses without neck are not skipped
Hi! Thank you for your work!

I explore the code and it seems that [this line](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation-3d-demo.pytorch/blob/master/modules/parse_poses.py#L36) which should check that the neck was found, does not do anything. It checks `x` coordinate of the keypoint to be equal to `-1`, but coordinates are already divided by `upsample_ratio` [few strings above](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation-3d-demo.pytorch/blob/master/modules/parse_poses.py#L29).
Seems it should check `z` coordinate of the keypoint.
The same goes for [this line](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation-3d-demo.pytorch/blob/master/modules/parse_poses.py#L100)

### PR DESCRIPTION
**Pull request by Daniil-Osokin**
Fixes #67

### CODE REVIEW

**Comment by Daniil-Osokin** [comment]
@platvlad Thanks, fixed!

**Comment by platvlad** [comment]
@Daniil-Osokin  Thank you!

File: modules/parse_poses.py [hunk 1]
```diff
@@ -33,11 +33,11 @@ def get_root_relative_poses(inference_results):
     num_kpt_panoptic = 19
     num_kpt = 18
     for pose_id in range(found_poses.shape[0]):
-        if found_poses[pose_id, 3] == -1:  # skip pose if does not found neck
+        if found_poses[pose_id, 2] == -1:  # skip pose if is not found neck
```
Comment (platvlad) [inline modules/parse_poses.py:36]: Seems it should be `if found_poses[pose_id, 5] == -1:`
This condition is before keypoint repacking. So now the pose is skipped if does not find nose

Comment (Daniil-Osokin) [reply modules/parse_poses.py]: You are right! Thanks, updated.

**Comment by Daniil-Osokin** [comment]
Great, thanks for the review!

**Comment by Daniil-Osokin** [comment]
:one: :hammer_and_pick: :accept:

**Pull request merged**